components/ButtonBar/src/MDCButtonBar.h

```diff
@@ -98,6 +98,19 @@ IB_DESIGNABLE
  */
 @property(nonatomic) CGFloat buttonTitleBaseline;
 
+/**
+ Sets the button title font for the given state.
```

Comment:
If a buttonBar could potentially have multiple Buttons then it might be better to term and talk about the setting of the font as Buttons (plural) or in the granularity of ButtonBar itself

Reply:
Done.